src/pages/questionPage.js

```diff
@@ -118,3 +119,24 @@ const showCorrectAnswer = () => {
     .classList.add('correct-answer');
 };
 
+let count = 0;
```

Comment:
We want to keep all non-dom logic in the views directory, so this variable and the related functions should go in questionView.js